docs/module-types.md

```diff
@@ -115,3 +116,61 @@ where `wasm-module.wasm` is generated from:
   )
 )
 ```
+
+## HTML Modules
+
+[HTML Modules](https://github.com/w3c/webcomponents/blob/gh-pages/proposals/html-modules-explainer.md) are html files whose [HTMLDocument](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDocument) is (by default) exposed to the javascript code importing the html module. The
+html from the module **is not inserted into the DOM** unless you do so after importing it.
+
+### Basic HTML module example
+```html
+<!-- module.html -->
+<template id="my-template">
+  A template
+</template>
+```
+
+```js
+System.import('module.html').then(function (module) {
+  const template = module.default.getElementById('my-template');
```

Comment:
Sorry, can you help me clarify - the getElById isn’t necessary here right? The default export (in this case) would just be the template too, right?

Reply:
No, the default export is an instance of HTMLDocument